label_studio/data_manager/models.py

```diff
@@ -9,6 +9,7 @@
 class ViewBaseModel(models.Model):
     data = models.JSONField(_('data'), default=dict, null=True, help_text='Custom view data')
     ordering = models.JSONField(_('ordering'), default=dict, null=True, help_text='Ordering parameters')
+    order = models.IntegerField(_('order'), default=0, help_text='Order of view')
```

Comment:
This migration isn't compatible with our migration guidelines - we shouldn't be adding new fields that aren't nullable, because they cause problems if we have to revert.

Reply:
While we're here, I think the help text could clarify the purpose of this field more; "order of view" could mean a lot of different things. We should spell out that it's the position of the tab, starting at the left in data manager and increasing as the tabs go left to right.

Reply:
Fixed.